app/forms/member_form.rb

```diff
@@ -0,0 +1,40 @@
+class MemberForm
+  include ActiveModel::Model
+  include Virtus
+
+  attribute :email, String
+  attribute :board_id, Integer
+
+  attr_reader :member_form
+
+ validates :email, presence: true, format: { with: /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/, message: 'incorrect email'}
+ validates :board_id, presence: true
+ validate :user?
+
+  def save
+    if valid?
+      persist!
+      true
+    else
+      false
+    end
+  end
+
+  private
+  def user?
+    user = User.find_by(email: email)
+    board = Board.find(board_id)
+    unless user
```

Comment:
I'm always avoid `unless else` because it's confusing.
So you can user you knowledge about method `find_by` and make this conditon ;)
`if user.nil`